content/en.json

```diff
@@ -143,16 +136,16 @@
       ]
     },
     "technology": {
-      "title": "Used Technology",
+      "title": "Technology",
       "subtitle": "We prefer using \"<em>the best tool for the job</em>\"; we're generalists who can quickly specialize when the need arises.",
       "how-title": "How we like to do this:",
       "how": [
-        "We do custom-made software; aiding in buy-over-build where needed",
-        "Hosting (usually) in the cloud",
+        "OS is a personal choice per developer",
```

Comment:
Noticed you slipstreamed an _addition_ into the PR about brevity, but seems like a good change to me still.